assets/csidriveroperators/vsphere/03_sa.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
```

Comment:
Using `03_sa.yaml` to be able to compare to / copy from the vsphere CSI driver operator `manifests/`